org.eclipse.xtext.xtext.ide/src-gen/org/eclipse/xtext/xtext/ide/contentassist/antlr/internal/InternalXtext.g

```diff
@@ -9874,9 +9874,9 @@ rule__EnumLiteralDeclaration__EnumLiteralAssignment_0
 	(
 		{ before(grammarAccess.getEnumLiteralDeclarationAccess().getEnumLiteralEEnumLiteralCrossReference_0_0()); }
 		(
-			{ before(grammarAccess.getEnumLiteralDeclarationAccess().getEnumLiteralEEnumLiteralIDTerminalRuleCall_0_0_1()); }
```

Comment:
wonder why i dont see any change in ide parser

Reply:
How do you generate the code?

Reply:
e.g. I use these args in the launch config:
```src/org/eclipse/xtext/xtext/bootstrap/GenerateXtext.mwe2 --param eclipseProjectPath=${project_loc:/org.eclipse.xtext.xtext.ui}```

I suspect that the IDE code was generated to another location.

Reply:
Something else: isn't this the IDE parser?

Reply:
i just ran GenerateXtext from eclipse

Reply:
ahhh it actually is

Reply:
:+1: